src/java.desktop/share/classes/javax/swing/text/DefaultCaret.java

```diff
@@ -366,6 +366,8 @@ protected void moveCaret(MouseEvent e) {
         }
     }
 
+    private int savedBlinkRate = 0;
+    private boolean isBlinkRateSaved = false;
     // --- FocusListener methods --------------------------
 
     /**
```

Comment:
I guess everybody noticed the specification of this method.
>This is implemented to set the caret to visible if the component is editable.

That should be updated? And overall it seems the whole behavior change required CSR.

Reply:
ping

Reply:
The documented behaviour does not change so no CSR is required. Documentation does not specify what happens when component is not editable it only mentions what this method does about the editable component. Amending the documentation on the other hand would require the CSR.